src/Assets/Animations.tsx

```diff
@@ -1,25 +1,21 @@
-import { css, keyframes } from "styled-components"
+import { keyframes } from "styled-components"
 
-export const fadeOut = css`
-  animation: ${keyframes`
-    from {
-      opacity: 1;
-    }
+export const fadeIn = keyframes`
+  from {
+    opacity: 0;
+  }
 
-    to {
-      opacity: 0;
-    }
-  `} 0.5s both;
+  to {
+    opacity: 1;
+  }
 `
 
-export const fadeIn = css`
-  animation: ${keyframes`
-    from {
-      opacity: 0;
-    }
+export const fadeOut = keyframes`
+  from {
+    opacity: 1;
+  }
 
-    to {
-      opacity: 1;
-    }
-  `} 0.5s both;
+  to {
+    opacity: 0;
+  }
```

Comment:
This animation stuff is a bit out-dated. Feel free to drop it entirely and add whatever makes more sense!

Reply:
Yeah I thought about it but I ended  updated it since it worked pretty well for the fadeIn/Out animation I needed to implement instead of using `react-addons-css-transition-group`